Nos/Views/Components/Media/LinkPreview.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import LinkPresentation
+import SDWebImageSwiftUI
+
+/// Shows an image button or a link view, depending on the URL.
+struct LinkPreview: View {
```

Comment:
One or two #Previews are missing here (I appreciate the Previews you wrote for the other views)